src/blocks/BulletPointBlock/BulletPointBlock.stories.tsx

```diff
@@ -0,0 +1,152 @@
+import { select, withKnobs, boolean } from '@storybook/addon-knobs'
```

Comment:
OK so this is nothing we should change now (but maybe add a ticket or plan for maintenance day?), but the `@storybook/addon-knobs` packages is apparently [deprecated](https://www.npmjs.com/package/@storybook/addon-knobs) so we should probably replace that.

Reply:
Yes indeed! @styrelius I tried to update storybook two times but always ran in to some babel error 😅 So maybe good to give it another stab tomorrow on maintenance day or just fix it in the new setup 👍